src/main/resources/application.properties

```diff
@@ -0,0 +1,6 @@
+db.driver="com.mysql.jdbc.Driver"
+db.host = "jdbc:mysql://localhost"
```

Comment:
instead write like this:
db.host = jdbc:mysql://localhost
and so on..

Reply:
ok made some changes.